test/integration/admin_area/account_activities_test.rb

```diff
@@ -36,4 +36,29 @@ def test_download_account_activity
                   response.headers['Content-Disposition']
     assert_not_empty response.body
   end
+
+  def test_search_account_activity
+    account_activities(:one).update(description: "Description of activity one", 
+                                    sum: "123.00",
+                                    activity_type: "create",
+                                    created_at: Time.zone.parse('2021-07-05 10:00')
+                                   )
```

Comment:
Could you, please, but ending bracket into the previous line?
Best practices are either 
```
(<some_args>,
<some_args>)
```
or
```
(
  <some_args>,
  <some_args>
)
```